src/loki/height_action.cc

```diff
@@ -79,12 +79,17 @@ std::vector<PointLL> loki_worker_t::init_height(Api& request) {
 }
 */
 std::string loki_worker_t::height(Api& request) {
+  // time this whole method and save that statistic
+  midgard::scoped_timer<> t([&request](const midgard::scoped_timer<>::duration_t& elapsed) {
+    auto e = std::chrono::duration_cast<std::chrono::duration<double, std::milli>>(elapsed).count();
+    auto* stat = request.mutable_info()->mutable_statistics()->Add();
+    stat->set_name("loki_worker_t::height");
+    stat->set_value(e);
+  });
```

Comment:
this block is the same throughout, when the timer goes out of scope this lambda gets called back with the time that passed since the object was created. then we set the name and the value. more of an explanation of this pattern can be read here: https://stackoverflow.com/questions/2808398/easily-measure-elapsed-time/55265585#55265585

Reply:
Could the variable be named something clearer like `timer` instead of `t`?